code/modules/antagonists/vampire/vampire_powers/gargantua_powers.dm

```diff
@@ -1,3 +1,5 @@
+#define ARENA_SIZE 3
```

Comment:
1. Is this being undefined later somewhere?

2. I am not sure about this myself but is it better to keep this close to the spell related to it instead of the beginning of the file?